server/app/src/androidTest/java/androidx/test/uiautomator/UiScrollableCustom.java

```diff
@@ -6,19 +6,27 @@ public UiScrollableCustom(UiSelector container) {
     }
 
     public boolean scrollIntoView(UiSelector selector) throws UiObjectNotFoundException {
-        Tracer.trace(new Object[]{selector});
-        UiSelector childSelector = this.getSelector().childSelector(selector);
-        boolean found = false;
+        return scrollIntoView(selector, 100);
+    }
```

Comment:
The replacement changes made here will not affect the existing behavior since this method is called by class "ClearNotificationPanel" and the class is specified with parent selector "UiSelector notificationStackScroller = new UiSelector().resourceId("com.android.systemui:id/notification_stack_scroller")" .. hence it will utilize the child selector option, instead of selector.